gpgopt.py

```diff
@@ -0,0 +1,106 @@
+import gnupg
+from pathlib import Path
+from os import getlogin
+from random import randint
+
+
+class GpgOpt():
+    """Initial gnupg, create, import, and export key pairs.
+
+    create, import, and export key pairs.
+    Also, can encrypt and decrypt message.
+    """
+
+    def __init__(self, mygnupghome='gnupgkeys', email=getlogin() + '@scmail.dev'):
```

Comment:
Because we've committed to using Python 3.6+ we're able to leverage [f-strings](https://www.python.org/dev/peps/pep-0498/) which I find more idiomatic than string concatenation.
```suggestion
    def __init__(self, mygnupghome='gnupgkeys', email=f'{getlogin()}@scmail.dev'):
```

Reply:
Thanks. I also changed other `.format` to f-string.